contents/product-engineers/trunk-based-development.md

```diff
@@ -0,0 +1,122 @@
+---
+title: How we do trunk-based development (and why you should too)
+date: 2024-01-19
+author: ["ian-vanagas"]
+rootpage: /blog
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Product engineers
+ - Engineering
+ - Feature management
+---
+
+What do trees, elephants, and successful software all have in common? They all have strong trunks helping them grow big and strong. In the case of software, the trunk is the main codebase engineers make changes to and that deploys to users. 
+
+There are many strategies for branching off and making changes to this trunk. The one we use at PostHog is trunk-based development. It helps us reduce work-in-progress, ship faster, keep momentum, and ultimately, build a more successful product. 
+
+This post covers what trunk-based development is, its benefits, why we do it at PostHog, and how you can implement it for yourself.
```

Comment:
Couple of thoughts:

- Could we make this intro more opinionated? Maybe talk about some of the benefits to us – talk up our shipping speed etc.
- Add more context about PostHog. Who are we? Why should the reader listen to us?

Additional... I feel like the reader hear is an engineer / engineering leader who want to enact change. How can we make the intro + headline speak to them?